ts/quickjs.ts

```diff
@@ -361,6 +361,13 @@ export class QuickJSVm implements LowLevelJavascriptVm<QuickJSHandle> {
       // we allocated a string
       quickJSKey.dispose()
     }
+
+    if (get) {
+      get.dispose()
+    }
+    if (set) {
+      set.dispose()
+    }
```

Comment:
`get` and `set` are always truthy because they're either a Lifetime object holding a pointer to a function (truthy because object is truthy), or a StaticLifetime object holding a pointer to undefined (truthy because object is truthy).

It's safe to write this instead as
```suggestion
    // dispose created functions; or no-op if these are this.undefined
    get.dispose()
    set.dispose()
```